contracts/test/strategies/frax-ETH.fork-test.js

```diff
@@ -21,34 +23,127 @@ forkOnlyDescribe("ForkTest: Frax ETH Strategy", function () {
     fixture = await loadFixture(fraxETHStrategyForkedFixture);
   });
 
-  describe.only("Mint", function () {
-    it("Should deploy fraxETH in Frax ETH Strategy", async function () {
-      const { daniel, frxETH } = fixture;
-      await mintTest(fixture, daniel, frxETH, "10");
-    });
+  it("Should deposit fraxETH in Frax ETH Strategy", async function () {
+    const { daniel, frxETH } = fixture;
```

Comment:
frxETH isn't used here. Through this file there's a few things lint complains about.

Reply:
thanks fixef